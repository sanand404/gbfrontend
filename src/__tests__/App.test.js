import React from "react";
import {shallow} from 'enzyme';
import AppComponent from '../../src/App'; 
let wrapper
beforeEach(()=>{
wrapper = shallow(<AppComponent/>)
})
describe('Testing App',()=>{
    it('Render as expected',()=>{
                expect(wrapper).toMatchSnapshot();
    })
    test('',()=>{})
})