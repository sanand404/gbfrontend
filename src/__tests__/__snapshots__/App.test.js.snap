// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Render as expected 1`] = `
<div>
  <h1>
    Hi Shradha
  </h1>
</div>
`;
